onnx_tf/frontends/frontend_v1.py

```diff
@@ -225,6 +225,9 @@ def handle_squeeze(cls, node, **kwargs):
   @classmethod
   @register_onnx_op("Tile")
   def handle_tile(cls, node, **kwargs):
+    if kwargs["node_dict"][node.inputs[1]].attr["dtype"] != TensorProto.INT64:
```

Comment:
You can always set a data_type_cast_map here because you checked if the src data type and target data type are the same in your casting logic.

So you don't really need this "if" here.

Reply:
lol... I forgot my own codes....